controller/array_action/array_mediator_xiv.py

```diff
@@ -163,6 +183,41 @@ def delete_volume(self, volume_id):
 
         logger.info("Finished volume deletion. id : {0}".format(volume_id))
 
+    def get_snapshot(self, snapshot_name):
+        logger.debug("Get snapshot : {}".format(snapshot_name))
+        try:
+            cli_snapshot = self.client.cmd.vol_list(vol=snapshot_name).as_single_element
+        except xcli_errors.IllegalNameForObjectError as ex:
+            logger.exception(ex)
+            raise controller_errors.IllegalObjectName(ex.status)
+        if not cli_snapshot:
+            return None
```

Comment:
so why did you define `SnapshotNotFoundError`?

Reply:
I had it in my [rev snapshot branch. Need it for next ticket
Removed for now

Reply:
but will you use this error here in the "revert to snapshot" ticket?
if so, please add its number here.
if not, then do you think we should return None (and in all mediators)?

Reply:
Here no. In CreateVolume
yes. We should return none in snapshot not found in all mediators